assets/prototype/domain/eventEditor/containers/mutations/prices/useOnUpdatePrice.js

```diff
@@ -0,0 +1,24 @@
+import useRelations from '../../../../../infrastructure/services/relations/useRelations';
+
+const useOnUpdatePrice = () => {
+	const { updateRelations } = useRelations();
+
+	const onUpdatePrice = ({ price, priceTypeId }) => {
+		const { id: priceId } = price;
+		// if it's not the optimistic response
+		// and priceType has been updated.
+		if (priceId && priceTypeId) {
```

Comment:
It's not entirely clear how this conditional is checking `if it's not the optimistic response`,
maybe create a variable with those conditions?

Reply:
An optimistic response won't have the `id` field if we create one.

Reply:
Oh yeah, forgot about that :man_facepalming: